core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.ts

```diff
@@ -496,15 +497,20 @@ export class StoryNodeEditorComponent implements OnInit, OnDestroy {
   updateLocalEditableOutline($event: string): void {
     if (this.editableOutline !== $event) {
       this.editableOutline = $event;
-      if (!this.chapterOutlineButtonsAreShown && $event) {
-        this.toggleChapterOutlineButtons();
-      }
       this.changeDetectorRef.detectChanges();
     }
   }
 
-  toggleChapterOutlineButtons(): void {
-    this.chapterOutlineButtonsAreShown = !this.chapterOutlineButtonsAreShown;
+  onSaveButtonClicked(): void {
+    this.oldOutlineToReadOnly = $(this.oldOutline).text();
```

Comment:
Please try to avoid use of jQuery here. 

Also, why do you need to do this? What does "oldOutlineToReadOnly" represent, exactly?

Reply:
Hi @seanlip. I use jQuery to remove html tag "<p></p>". I'm stuck on how to remove the tag. Could I  have some advice?